bin/count-daily-stats.js

```diff
@@ -1,8 +1,12 @@
 #!node_modules/.bin/babel-node
 import moment from 'moment';
 import pgFormat from 'pg-format';
+import cld from 'cld';
 
-import { postgres } from '../app/models';
+import { dbAdapter, postgres } from '../app/models';
+
+const langDAUCode = 'ru';
```

Comment:
@davidmz does this library work for Fārsī?

Reply:
Yes, it should: https://github.com/CLD2Owners/cld2#:~:text=Nepali%20Norwegian%20Oriya-,persian,-Polish%20Portuguese%20Punjabi (language code is "fa")

Reply:
Perhaps instead of fixing the language in the code, we should pass it as a CLI argument (maybe even a list of languages)? I don't know exactly how we run this script.